Assets/Scripts/PlayerControls.cs

```diff
@@ -9,18 +9,28 @@ public class PlayerControls : MonoBehaviour
 	[SerializeField] float _interactCheckRadius = 0.2f;
 
 	PlayerActor _actor;
-	ActorPhysics _actorPhysics;
-	Cutting _cutting;
 	
 	Button _holdButton = new Button( "Hold" );
+	public Button holdButton
+	{
+		get { return _holdButton; }
+	}
```

Comment:
I still feel like this is confusing especially in the context of buttons. Consider grasp, clutch or grip.